apps/design-land/src/app/app.component.scss

```diff
@@ -3,14 +3,27 @@
 }
 
 .design-land {
-  
   &__theme-switch {
     margin-left: auto;
   }
 
   &__content {
     padding: 24px 48px 48px;
   }
+
+  &__sidebar {
+    width: 288px;
+    border-right: 1px solid #e8e8e8;
```

Comment:
We do this border in a few places. Can we embed this in design somehow?

Reply:
No. This design decision should be up to the end user.